execution_engine_testing/tests/src/test/regression/gh_1931.rs

```diff
@@ -0,0 +1,77 @@
+use casper_engine_test_support::{
+    DeployItemBuilder, ExecuteRequestBuilder, InMemoryWasmTestBuilder, ARG_AMOUNT,
+    DEFAULT_ACCOUNT_ADDR, DEFAULT_PAYMENT, DEFAULT_RUN_GENESIS_REQUEST,
+};
+use casper_types::{runtime_args, CLValue, Key, RuntimeArgs};
+use core::convert::TryFrom;
+
+const CONTRACT_SET_NAMED_KEY_STORED: &str = "set_named_key_stored.wasm";
```

Comment:
make these constant names less horrible

Reply:
Is the file `set_named_key_stored.wasm` missing from the PR? I can't run this test due to:
```
thread 'test::regression::gh_1931::main' panicked at '
Failed to open compiled Wasm file.  Tried the following locations:
    - /home/magister/Casper/casper-node/target/wasm32-unknown-unknown/release/set_named_key_stored.wasm
    - /home/magister/Casper/casper-node/execution_engine_testing/tests/wasm/set_named_key_stored.wasm
    - set_named_key_stored.wasm
```

Reply:
Unresolving due to the new comment for the same line.

Reply:
You need to `make build-contracts-rs` to make sure that the compiled WASM file exists.